libs/dfi/include/avrobin_serializer.h

```diff
@@ -32,12 +32,65 @@ extern "C" {
 //logging
 DFI_SETUP_LOG_HEADER(avrobinSerializer);
 
+/**
+ * @brief Deserialize data in AVRO format.
+ *
+ * The caller is the owner of the deserialized data and use `dynType_free` deallocate the memory.
+ *
+ * In case of a error, an error message is added to celix_err.
```

Comment:
```suggestion
 * In case of an error, an error message is added to celix_err.
```
Typo. I guess this is not the only one.